### ISSUE TITLE
[BUG] : Tap on line with webgl breaks java script

### ISSUE THREAD

**Issue by FloBay** [bokeh/bokeh#13341]
### Software versions

Python version      :  3.10.9 | packaged by Anaconda, Inc. | (main, Mar  1 2023, 18:18:15) [MSC v.1916 64 bit (AMD64)]
IPython version     :  (not installed)
Tornado version     :  6.2
Bokeh version       :  3.0.3
BokehJS static path :  \.conda\envs\curve_finder\lib\site-packages\bokeh\server\static
node.js version     :  (not installed)
npm version         :  (not installed)
Operating system    :  Windows-10-10.0.19045-SP0

### Browser name and version

Firefox v116.0.2

### Expected behavior

When using the tap tool and clicking on the line (only in wegbl mode ! canvas is fine). The plot dissapears and JavaScript reports an error. Only refreshing the page can bring back the plot correctly.

 

### Observed behavior

JS Error only in the webgl mode

### Example code

```Python
import numpy as np
import bokeh
from bokeh.models import ColumnDataSource
from bokeh.plotting import figure, output_file, save
from bokeh.layouts import layout

# Version
print(bokeh.__version__)

# Data
line = ColumnDataSource({'x': np.arange(-5, 5), 'y': np.arange(-5, 5)})

# Figure
tools = "tap"
backend = 'webgl'  # problematic
#backend = 'canvas'  # works
fig = figure(width=700, height=600, tools=tools, toolbar_location="above", output_backend=backend)
line = fig.line(x='x', y='y', line_width=2, source=line, color='red', line_dash='dashed')

# Export
grid = layout([fig])
output_file(filename=f"./error_test_{backend}.html", title="test")
save(fig)
```

### Stack traceback or browser console output

![image](https://github.com/bokeh/bokeh/assets/14998394/c23ddf07-b305-4383-b0ff-241b12bafbc5)

### Screenshots

Before the tab:
![image](https://github.com/bokeh/bokeh/assets/14998394/ad39ec2e-3da6-47fd-a840-1ce4dd2bdf8e)
After the tab on the red line:
![image](https://github.com/bokeh/bokeh/assets/14998394/1aa3785a-3682-4fd9-9d65-99c1a607ec8c)

**Comment by bryevdv** [comment]
@FloBay there have been a number of webgl-improvements lately so please first try to reproduce any issue with latest Bokeh version 3.2.2 and report back. 

cc @ianthomas23

**Comment by FloBay** [comment]
@bryevdv my bad. I was in the false enviroment. But still we have the same problem with version 3.2.2 too 

Python version      :  3.10.9 | packaged by Anaconda, Inc. | (main, Mar  1 2023, 18:18:15) [MSC v.1916 64 bit (AMD64)]
IPython version     :  (not installed)
Tornado version     :  6.2
Bokeh version       :  3.2.2
BokehJS static path :  \.conda\envs\curve_finder_2\lib\site-packages\bokeh\server\static
node.js version     :  (not installed)
npm version         :  (not installed)
Operating system    :  Windows-10-10.0.19045-SP0

![image](https://github.com/bokeh/bokeh/assets/14998394/a17eb5eb-1a00-4856-828d-536a404da96d)

**Comment by ianthomas23** [comment]
This is a duplicate of issue #13195. It is fixed in `branch-3.3` and will be in the next release.
